package.json

```diff
@@ -21,7 +21,7 @@
     "compute-cluster": "0.0.9",
     "convict": "2.0.0",
     "fxa-notifier-aws": "1.0.0",
-    "gm": "1.22.0",
+    "gm-reloaded": "1.24.0",
```

Comment:
something exploded in the tests, will need to debug. I don't see the much in this diff https://github.com/vladikoff/gm-reloaded/compare/1.22.0...v1.24.0